File: NeoTech (Aplicação Windows)/NeoTech/ClassesAuxiliares/UnderZeroException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NeoTech.ClassesAuxiliares
{
    class UnderZeroException : Exception
    {
        public override string ToString() { return "Valor menor que zero detectado"; }
    }
}

File: NeoTechApp/app/src/main/java/com/example/familiapaiva/neotechapp/UnderZeroException.java
package com.example.familiapaiva.neotechapp;

/**
 * Created by Familia Paiva on 12/12/2017.
 */

public class UnderZeroException extends Exception {
    @Override
    public String toString() {
        return "Valor menor que zero detectado";
    }
}
